internal/store/driver.go

```diff
@@ -379,10 +389,12 @@ func (d *Driver) calculateHTTPSEdges() []ingressv1alpha1.HTTPSEdge {
 		for _, ingress := range ingresses {
 			namespace := ingress.Namespace
 
+			module_load_failure := false
 			modSet, err := d.getNgrokModuleSetForIngress(ingress)
 			if err != nil {
 				d.log.Error(err, "error getting ngrok moduleset for ingress", "ingress", ingress)
-				continue
+				module_load_failure = true
+				d.recorder.Event(&edge, corev1.EventTypeWarning, "NgrokModuleLoadFailure", err.Error())
```

Comment:
we are just loading the `NgrokModuleSet` objects from the k8s api, this feels like the wrong place to put this. The right place should be somewhere in `httpsedge_controller.go`, where it tries to set the modules for and edge/route.

Reply:
You're referring to the recorder event?

Reply:
Yeah, on a high level, I'm referring to the fact that its not an issue that this fails, but the reconciliation process that happens over the httpsedges later on is what will error/we need to log

Reply:
Got it 👍🏼 I think moving it to the edge reconciler will also let us catch other initialization failures too.